src/components/Card/Card.scss

```diff
@@ -157,11 +157,11 @@ $block: '.#{variables.$ns}card';
 
     &_size {
         &_m {
-            --_--border-radius: 8px;
+            --_--border-radius: var(--g-border-radius-l);
```

Comment:
Why we use `--g-border-radius-l` for `size_m`?

Reply:
Card's border sizes don't have straight accordance to radius constants. I have choosen constants by values.
https://github.com/gravity-ui/uikit/blob/main/styles/themes/common/_index.scss#L13

Reply:
@amje I think it is a not good idea to use symantic variables in such cases

Reply:
What do you think about adding variables for cards? 
For example:
—g-border-radius-card-m
—g-border-radius-card-l

Reply:
> What do you think about adding variables for cards to _index.scss?
> For example:
> —g-border-radius-card-m
> —g-border-radius-card-l

@korvin89 , @amje, I mean like this: https://github.com/gravity-ui/uikit/pull/1682/commits/a50b4604eade6bc6ab47d30fe7d86ceadc100ed6

In my opinion, these variables also helps to make themisation easier, without using CSS API